HACK fix early console for iMX-35

This patch makes the MXC_LL* defines known at the location we need them.
A better location has to be found, but it works for now.

Signed-off-by: Remy Bohmer <linux@bohmer.net>
---
 include/asm-arm/arch-mxc/mx35.h |    3 +++
 1 file changed, 3 insertions(+)

Index: linux-2.6.26/include/asm-arm/arch-mxc/mx35.h
===================================================================
--- linux-2.6.26.orig/include/asm-arm/arch-mxc/mx35.h	2009-03-16 23:24:04.000000000 +0100
+++ linux-2.6.26/include/asm-arm/arch-mxc/mx35.h	2009-03-16 23:26:49.000000000 +0100
@@ -452,4 +452,7 @@
 #define NFMS_NF_DWIDTH		14
 #define NFMS_NF_PG_SZ		8
 
+#define MXC_LL_UART_PADDR       UART1_BASE_ADDR
+#define MXC_LL_UART_VADDR       AIPS1_IO_ADDRESS(UART1_BASE_ADDR)
+
 #endif				/*  __ASM_ARCH_MXC_MX35_H__ */
